community/cypher/cypher/src/main/scala/org/neo4j/cypher/CypherException.scala

```diff
@@ -152,3 +152,5 @@ class LoadExternalResourceException(message: String, cause: Throwable) extends C
 class LoadCsvStatusWrapCypherException(extraInfo: String, cause: CypherException) extends CypherException(s"${cause.getMessage} (${extraInfo})", cause) {
   val status = cause.status
 }
+
+class ShortestPathFallbackDisabledException(message: String, cause: Throwable) extends CypherExecutionException(message, cause)
```

Comment:
Yeah the name is not the best, I have named it after the configuration name since I have failed to find anything better.